lab-doug/server.js

```diff
@@ -0,0 +1,57 @@
+const http = require('http');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const cowsay = require('cowsay');
+const getContent = require(__dirname + '/lib/getContent');
+
+const port = process.argv[2] || 3000;
+
+const server = http.createServer(function(req, res){
+  //convert from string to object
+  req.url = parseUrl(req.url);
+  /*convert a query string to an object within the req.url object*/
+  req.url.query = parseQuery(req.url.query);
+
+  if(req.url.pathname == '/'){
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    res.write('API Endpoints:/api/cowsay');
+    res.end();
+  }
+  if(req.url.pathname == '/cowsay' && req.method == 'GET' ) {
+    //http localhost:3000/cowsay 'text==saywat'
+    if (req.url.query['text'] === 'saywat'){
```

Comment:
We're trying to write a server that can let people get the cow to say anything they want. However, this code only allows them to ask the cow to say 'saywat'. See if you can refactor this so that the cow can say more things.